src/index.html

```diff
@@ -5,9 +5,54 @@
     <meta name="viewport"
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <title>Product cards</title>
-    <link rel="stylesheet" href="./styles/index.scss">
+    <link rel="stylesheet" href="../src/styles/index.scss">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div
+      class="product"
+      data-qa="card"
+    >
+      <img
+        class="product__image"
+        src="./images/imac.jpeg"
+        alt="product__image"
+      >
+
+      <p class="product__description">
+        APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+      </p>
+
+      <div class="product__code">
+        Product code: 195434
+      </div>
+
+      <div class="card">
+        <div class="stars">
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+        </div>
+
+        <p class="card__review">Reviews: 5</p>
+      </div>
+
+      <div class="price">
+        <p class="price__name">Price:</p>
+
+        <p class="price__amount">$2,199</p>
+      </div>
```

Comment:
it's an elements of `product` block
```suggestion
      <div class="product__price">
        <p class="product__price_name">Price:</p>
        <p class="product__price_amount">$2,199</p>
      </div>
```